MANIFEST.in

```
@@ -3,9 +3,7 @@
 # in production. Note that this package is generated
 # by running 'python setup.py sdist' in build.py.
 
-include assets/constants.ts
-include assets/release_constants.json
-include assets/rich_text_components_definitions.ts
+include assets
```

```suggestion
graft assets
```

See the notation here https://packaging.python.org/en/latest/guides/using-manifest-in/#manifest-in-commands

---

Done